pkg/cloud/services/awsnode/cni.go

```diff
@@ -51,6 +52,11 @@ func (s *Service) ReconcileCNI() error {
 		return err
 	}
 
+	metaLabels := map[string]string{
```

Comment:
Is there a CAPI way to mark workload cluster resources as owned by some management cluster?

Reply:
Nothing that I've been able to find.

Reply:
@randomvariable do you have any input on this at all? It's the only outstanding thing.